### ISSUE TITLE
Allow training without a test set?

### ISSUE THREAD

**Issue by hmacdope** [OpenADMET/openadmet-models#177]
After marinating on this more, it does make sense to not require. We'll have a val split for hyperparameter tuning, but once selected we should be able to train on all data. Along these lines, should we be able to train without test as well? I.e. full production mode?

_Originally posted by @smcolby in https://github.com/OpenADMET/openadmet-models/pull/175#discussion_r2112468182_

**Comment by hmacdope** [comment]
@smcolby we made this change no? Re-open if not.
